extensions/svelte/extension.toml

```
@@ -1,7 +1,7 @@
 id = "svelte"
 name = "Svelte"
 description = "Svelte support"
-version = "0.1.1"
+version = "0.2.0"
```

We will bump the version separately:

```suggestion
version = "0.1.1"
```